src/tests/ftest/io/fio_small.yaml

```diff
@@ -32,13 +32,13 @@ server_config:
       scm_list: ["/dev/pmem1"]
       scm_mount: /mnt/daos1
   transport_config:
-    allow_insecure: True
```

Comment:
Test with secure mode was failed before.
So I keep the "transport_config: 
                         allow_insecure: False "
instead of delete it.